monsieurbiz/sylius-alert-message-plugin/1.0/config/packages/monsieurbiz_sylius_alert_message_plugin.yaml

```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: "@MonsieurBizSyliusAlertMessagePlugin/Resources/config/config.yaml" }
```

Comment:
This looks strange. What is happening herer?

Reply:
What is strange?

The fact that it's not a `Bundle` but a `Plugin`?

See -> https://github.com/symfony/flex/pull/426

Reply:
I find it strange that you need to add a file that only imports a config file from your plugin. 

Why dont you include that file [here](https://github.com/monsieurbiz/SyliusAlertMessagePlugin/blob/master/src/DependencyInjection/MonsieurBizSyliusAlertMessageExtension.php#L21) just as you do with `services.yaml`?

Reply:
Because I use the SyliusResourceBundle. And if I don't do it this way I get the following error:

`There is no extension able to load the configuration for "sylius_resource"…`

So… 🤷‍♂️

Reply:
You get that error because a bundle cannot modify other bundles config. Like a recipe should not modify other bundles config. You are trying to hide config to other bundles by using this import. 

=)

That is why I think this is weird. 

--------

I see that you are adding to the `grid` collection in the GridBundle and similarly adding to a collection in the Resource bundle. That is fine to have in a recipe.

Reply:
It's a merge then! 🥰

Reply:
Or did I miss something @Nyholm?

Reply:
It took me a while to be okey with this Sylius thing. Sorry for the wait.

Reply:
No harm done! Thank you!